src/node_crypto.cc

```diff
@@ -211,6 +211,95 @@ static int NoPasswordCallback(char* buf, int size, int rwflag, void* u) {
 }
 
 
+namespace error {
+Local<Object> DecorateO(Environment* env, Local<Object> obj, unsigned long err) {
+  if (err == 0) return obj;  // No decoration possible.
+
+  const char* ls = ERR_lib_error_string(err);
+  const char* fs = ERR_func_error_string(err);
+  const char* rs = ERR_reason_error_string(err);
+
+  obj->Set(env->isolate()->GetCurrentContext(),
+      OneByteString(env->isolate(), "library"),
```

Comment:
These can be made static strings stored in `env`?

Reply:
Yes, I'll do that.